fuzz/__main__.py

```diff
@@ -13,83 +13,129 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# import hashlib
+import hashlib
+import logging
 import os
 import sys
-import atheris
+from unittest.mock import patch, MagicMock
 
+import atheris
 # # # In simple case interested lib(s) may be imported during 'with'
 # # # It runs quickly but not precisely
 # with atheris.instrument_imports(enable_loader_override=False):
-import requests.exceptions
-
-import credsweeper
+import requests
+from google_auth_oauthlib.flow import InstalledAppFlow
 from oauthlib.oauth2 import InvalidGrantError
 from requests import Response
-from unittest.mock import patch, MagicMock
 
-cred_sweeper = credsweeper.app.CredSweeper()
+from credsweeper import CredSweeper, DataContentProvider, ApplyValidation
+from credsweeper.validations import GithubTokenValidation, GoogleApiKeyValidation, MailChimpKeyValidation, \
+    StripeApiKeyValidation, SquareClientIdValidation, SlackTokenValidation, SquareAccessTokenValidation, \
+    GoogleMultiValidation
+
+# set log level for fuzzing
+logging.basicConfig(level=logging.CRITICAL)
+logger = logging.getLogger(__name__)
+
+# need to set depth=1 to remove .zip and .gz from extension
+cred_sweeper = CredSweeper(depth=1)
+api_validation = ApplyValidation()
+
+INPUT_DATA_SIZE = 0x0600
+BEHAVIOUR_BYTE_SIZE = 0x01
+MOCK_RESPONSE_SIZE = 0x01FF
+
+
+def mock_request(behaviour_code: int, status_code_seed: int, content: bytes, candidate, patch_object, path_name):
```

Comment:
Can you please describe changes been done here, and how they are relates to gzip files?

Reply:
New code has reduced percent of dynamic testing (fuzzing) - i had to refuzzed but found that something might be optimized... #189